main.tf

```diff
@@ -32,13 +34,13 @@ variable "force_destroy" {
 # bucket for storing tf state
 resource "aws_s3_bucket" "bucket" {
   bucket        = "tf-state-${var.application}"
-  force_destroy = "${var.force_destroy}"
+  force_destroy = var.force_destroy
 
   versioning {
     enabled = "true"
```

Comment:
This should probably be `enabled = true` since 0.12 is more type strict (this burned us in another case).

Reply:
Thanks! Fixed.